tests/Unit/Database/Seeders/ProviderSeederTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\LoyaltyCorp\Multitenancy\Unit\Database\Seeders;
+
+use LoyaltyCorp\Multitenancy\Database\Entities\Provider;
+use LoyaltyCorp\Multitenancy\Database\Seeders\ProviderSeeder;
+use Tests\LoyaltyCorp\Multitenancy\Stubs\Vendor\Doctrine\EntityManagerStub;
+use Tests\LoyaltyCorp\Multitenancy\TestCase;
+
+/**
+ * @covers \LoyaltyCorp\Multitenancy\Database\Seeders\ProviderSeeder
+ */
+class ProviderSeederTest extends TestCase
+{
+    /**
+     * Tests that the Provider Seeder works as expected.
+     *
+     * @return void
+     */
+    public function testSeed(): void
+    {
+        $expected = [
+            new Provider('loyalty-corp', 'Loyalty Corp'),
+            new Provider('moon-corp', 'Mooncorp')
+        ];
+
+        $entityManager = new EntityManagerStub();
+        $seeder = new ProviderSeeder($entityManager);
+
+        $seeder->seed();
+
+        static::assertEquals($expected, $entityManager->getPersisted());
```

Comment:
If we need to make sure each line is tested, do we also want to make sure we assert flush was called?